Update graphing details for new metric dropdown (#11061)
DataDog/documentation: content/en/dashboards/querying/_index.md

### Choose the metric to graph

> Choose the metric to graph by searching or selecting it from the dropdown next to **Metric**. If you don't know which metric to use, the metric dropdown also provides additional information, including the `unit`, `type`, `interval`, `description`, `tags`, and number of `tag's values`. 

Reviewer: ```suggestion
Choose the metric to graph by searching or selecting it from the dropdown next to **Metric**. If you don't know which metric to use, the metric dropdown provides additional information, including the `unit`, `type`, `interval`, `description`, `tags`, and number of `tag values`. 
```

> {{< img src="dashboards/querying/metric_dropdown.png" alt="Metric Selector Dropdown"  style="width:75%;" >}}

Reviewer: ```suggestion
{{< img src="dashboards/querying/metric_dropdown.png" alt="Metric Selector Dropdown" responsive="true" style="width:75%;">}}
```

> If you want to explore your metrics further, start with the [Metrics Explorer][3] or a [notebook][4]. You can also see a list of metrics on the [Metrics Summary page][5].

Reviewer: ```suggestion
Explore your metrics further with the [Metrics Explorer][3], a [Notebook][4], or see a list of metrics on the [Metrics Summary page][5].
```
